openedx-scalable-ubuntu/azuredeploy.json

```diff
@@ -31,170 +31,199 @@
+    "loadBalancerName": "[concat(variables('namingInfix'), 'lb')]",
+    "publicIPAddressID": "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicIPAddressName'))]",
+    "lbID": "[resourceId('Microsoft.Network/loadBalancers',variables('loadBalancerName'))]",
+    "natPoolName": "[concat(variables('namingInfix'), 'natpool')]",
+    "bePoolName": "[concat(variables('namingInfix'), 'bepool')]",
+    "natStartPort": 50000,
+    "natEndPort": 50119,
+    "natBackendPort": 22,
+    "nicName": "[concat(variables('namingInfix'), 'nic')]",
+    "ipConfigName": "[concat(variables('namingInfix'), 'ipconfig')]",
+    "frontEndIPConfigID": "[concat(variables('lbID'),'/frontendIPConfigurations/loadBalancerFrontEnd')]",    
+    "lbPoolID": "[concat(variables('lbID'),'/backendAddressPools/', variables('bePoolName'))]",
     "lbProbeLMSID": "[concat(variables('lbID'),'/probes/tcpProbeLMS')]",
     "lbProbeCMSID": "[concat(variables('lbID'),'/probes/tcpProbeCMS')]",
-    "frontEndIPConfigID": "[concat(variables('lbID'),'/frontendIPConfigurations/loadBalancerFrontend')]"
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+    "subnetID": "[concat(variables('vnetID'),'/subnets/',variables('subnetName'))]",
+    "mongoVmName": "openedx-mongo",
+    "mongoVhdContainerName": "[concat(variables('mongoVmName'), 'vhd')]",
+    "mongoPublicIPAddressName": "[concat(variables('mongoVmName'), 'pip')]",
+    "mongoOSDiskName": "[concat(variables('mongoVmName'), 'osdisk')]",
+    "mySqlVmName": "openedx-mysql",
+    "mySqlVhdContainerName": "[concat(variables('mySqlVmName'), 'vhd')]",
+    "mySqlPublicIPAddressName": "[concat(variables('mySqlVmName'), 'pip')]",
+    "mySqlOSDiskName": "[concat(variables('mySqlVmName'), 'osdisk')]",
+    "scriptDownloadUri": "https://raw.githubusercontent.com/azure/azure-quickstart-templates/master/openedx-scalable-ubuntu/",
+    "installScript": "install-openedx.sh",
+    "mongoInstallCommand": "[concat('bash -c ''nohup ./', variables('installScript'), ' mongo  </dev/null &>/var/log/azure/install-openedx-mongo.log''')]",
+    "mySqlInstallCommand": "[concat('bash -c ''nohup ./', variables('installScript'), ' mysql  </dev/null &>/var/log/azure/install-openedx-mysql.log''')]",
+    "appInstallCommand":   "[concat('bash -c ''nohup ./', variables('installScript'), ' edxapp </dev/null &>/var/log/azure/install-openedx-app.log &''')]",
+    "dbStorageAccountName": "[concat(variables('uniqueStringArray')[4], variables('newStorageAccountSuffix'))]",
+    "diagnosticsStorageAccountName": "[concat(variables('uniqueStringArray')[0], variables('newStorageAccountSuffix'))]",
+    "diagnosticsStorageAccountId": "[concat('/subscriptions/',subscription().subscriptionId,'/resourceGroups/',resourceGroup().name,'/providers/','Microsoft.Storage/storageAccounts/', variables('diagnosticsStorageAccountName'))]",
+    "wadlogs": "<WadCfg><DiagnosticMonitorConfiguration>",
```

Comment:
this can be simplified a lot with host metrics (going forward, using host metrics instead of WAD/LAD is the recommended way autoscale on scale sets, at least for basic metrics). The changes are pretty simple: remove the WAD/LAD config and change the metric name; example PR doing this here: https://github.com/Azure/azure-quickstart-templates/pull/2723